log

```diff
@@ -0,0 +1,343 @@
+--- FAIL: TestPrivateWeave (0.88s)
+	integration_test.go:181: diff:
+		+ resource "aws_autoscaling_attachment" "bastions-bastions-privateweave-example-com" {
+		- resource "aws_autoscaling_attachment" "bastion-privateweave-example-com" {
+		    elb = "${aws_elb.bastion-privateweave-example-com.id}"
+		+   autoscaling_group_name = "${aws_autoscaling_group.bastions-bastions-privateweave-example-com.id}"
+		-   autoscaling_group_name = "${aws_autoscaling_group.bastion-privateweave-example-com.id}"
+		  }
+		  
+		...
+		  }
+		  
+		+ resource "aws_autoscaling_group" "bastions-bastions-privateweave-example-com" {
+		- resource "aws_autoscaling_group" "bastion-privateweave-example-com" {
+		+   name = "bastions.bastions.privateweave.example.com"
```

Comment:
It is, but it is consistent.  The instance group is called "bastions", the type of the instance group is "bastions".  I guess we _could_ just give it the name of the instance group - no type - because instance group names are unique, but we would still need to keep the name for masters for backcompat (nodes are not qualified).

(e.g. masters are `master-us-west-1a-masters-private-awsdata-com`)

I'll make this change :-)